hnn_core/params.py

```diff
@@ -246,6 +249,10 @@ def _extract_drive_specs_from_hnn_params(params, cellname_list):
 
             drive['weights_nmda'] = dict()  # no NMDA weights for Gaussians
         elif feed_name.startswith('extpois'):
+            if (not legacy_mode) and par['t_interval'][1] < par[
+                    't_interval'][0]:
+                print(f"interval!!!!! {par['t_interval']}")
```

Comment:
Were you wanting to keep this print statement or possibly convert it to a warning?

Reply:
the number of exclamation marks :rofl:

Reply:
Let's just say the GUI tests print A LOT output, need to make it distinct :wink: